### ISSUE TITLE
A/B test: Search notebook examples

### ISSUE THREAD

**Issue by rrhyne** [sourcegraph/sourcegraph#23096]
### Test

Hypothesis, search notebooks will show the unique value propositions of Sourcegraph search, which will increase signups and retention.
  
### Metric and experimental design
  
**Metric:** Retention
**Smaller significant change:** Current baseline is 10.4%, if retention is improved by 12%, thus reaches 13.9% the test is a success. 

**Significance threshold:** 5%
**Duration/size:** 1 week or 7,200 users
  
  
#### Descriptive analytics

* Impact on conversion ratio of new visitors to sign-ups. 
* Percentage of traffic clicks on a notebook vs. example searches
* Search success funnel: Homepage -> search results -> search success
  
### Flag

AB-test/notebook-examples

**Comment by rrhyne** [comment]
@malomarrec, please review!

**Comment by malomarrec** [comment]
Sounds good!
@ebrodymoore the notebook says
> Fill out all the following information in an issue with A/B testing as a linked project
https://about.sourcegraph.com/handbook/ops/bizops/ab-testing

I remember we said that we should create a project but I didn't find it. Created it here:
https://github.com/orgs/sourcegraph/projects/192

**Comment by malomarrec** [comment]
@rrhyne @novoselrok do we already have the metrics to track "Percentage of traffic clicks on a notebook vs. example searches"?
Do we already have `HomepageNotebookClicked`?

**Comment by rrhyne** [comment]
Good call. 

We already have example search metrics defined. I believe they were added in the build. ([Figma](https://www.figma.com/file/5CRqR5RVgv0HJ5A06YQ1uT/?node-id=95%3A3196))

I've added a metrics page for Notebooks just now: ([Figma](https://www.figma.com/file/g7CKzz87YtlkVdP4AaEKYT/?node-id=155%3A3250))

Note that the name of the event describes which example was clicked. If you change the content of the examples (I personally would not to make sure the tests are as apples to apples as possible), then you would need to change the labels and descriptions.
